### PR TITLE
Bump JuliaOpt packages for new informational callbacks

### PR DESCRIPTION
**Pull request by joehuchette**
Ref https://github.com/JuliaOpt/JuMP.jl/pull/814

### CODE REVIEW

**Comment by joehuchette** [comment]
@mlubin @chriscoey @yeesian

**Comment by tkelman** [comment]
these all need git tags to be pushed

File: MathProgBase/versions/0.5.6/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+119129282ab5b5c95c66d667c46b202aeb4e3f08
```
Comment (tkelman) [inline MathProgBase/versions/0.5.6/sha1]: this is the same as 0.5.5

Comment (joehuchette) [inline MathProgBase/versions/0.5.6/sha1]: Good point, this change only touched the docs for MathProgBase, and I guess that commit got lumped into an earlier release

**Comment by mlubin** [comment]
Thanks @joehuchette. NEWS update for the release?

**Pull request merged**